certbot/CHANGELOG.md

```diff
@@ -14,7 +14,12 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Fixed
 
-*
+* `IPluginFactory`, `IPlugin`, `IAuthenticator` and `IInstaller` have been re-added to
+  `certbot.interfaces`.
+    - This is to fix compatibility with a number of third-party DNS plugins which may
+      have started erroring with `AttributeError` in Certbot v2.0.0.
+    - Plugin authors can find more information about Certbot 2.x compatibility
+      [here]().
```

Comment:
Reminder for us to add a link here.

Do you think it's worth putting this on [the wiki](https://github.com/certbot/certbot/wiki) or [static docs](https://eff-certbot.readthedocs.io/en/stable/) and linking there to simplify this or do you think making the community forum post early is OK?

Reply:
Wiki sounds like a good place to put it, actually! I didn't particularly want to post this on the forums.